<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DcpTermWin</class>
 <widget class="QMainWindow" name="DcpTermWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DCP Terminal</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPlainTextEdit" name="textOutput">
      <property name="font">
       <font>
        <family>Monospace</family>
       </font>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="undoRedoEnabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="comboDevice">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="maxCount">
         <number>100</number>
        </property>
        <property name="insertPolicy">
         <enum>QComboBox::InsertAlphabetically</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboMessage">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="maxCount">
         <number>1000</number>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToMinimumContentsLengthWithIcon</enum>
        </property>
        <property name="duplicatesEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonSend">
        <property name="text">
         <string>Send</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>611</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuConnection">
    <property name="title">
     <string>&amp;Connection</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionAutoReconnect"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuOutput">
    <property name="title">
     <string>&amp;Output</string>
    </property>
    <addaction name="actionClearOutput"/>
    <addaction name="actionVerboseOutput"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuConnection"/>
   <addaction name="menuOutput"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionConnect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Connect</string>
   </property>
   <property name="toolTip">
    <string>Connect to DCP server</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="toolTip">
    <string>Quit the program</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About...</string>
   </property>
  </action>
  <action name="actionAutoReconnect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto &amp;Reconnect</string>
   </property>
   <property name="toolTip">
    <string>Reconnect automatically</string>
   </property>
  </action>
  <action name="actionVerboseOutput">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Verbose Output</string>
   </property>
   <property name="toolTip">
    <string>Verbose output</string>
   </property>
  </action>
  <action name="actionClearOutput">
   <property name="text">
    <string>Clea&amp;r Output</string>
   </property>
   <property name="toolTip">
    <string>Clear output text</string>
   </property>
  </action>
  <action name="actionClearDeviceList">
   <property name="text">
    <string>Clear Device List</string>
   </property>
   <property name="toolTip">
    <string>Clear the device list</string>
   </property>
  </action>
  <action name="actionClearMessageHistory">
   <property name="text">
    <string>Clear Message History</string>
   </property>
   <property name="toolTip">
    <string>Clear the entered messages</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>&amp;Settings...</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>comboMessage</tabstop>
  <tabstop>buttonSend</tabstop>
  <tabstop>textOutput</tabstop>
  <tabstop>comboDevice</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>DcpTermWin</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionClearOutput</sender>
   <signal>triggered()</signal>
   <receiver>textOutput</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionClearDeviceList</sender>
   <signal>triggered()</signal>
   <receiver>comboDevice</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>64</x>
     <y>483</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionClearMessageHistory</sender>
   <signal>triggered()</signal>
   <receiver>comboMessage</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>335</x>
     <y>483</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
